static/bidder-params/visx.json

```diff
@@ -5,7 +5,8 @@
   "type": "object",
   "properties": {
     "uid": {
-      "type": "integer",
+      "type": ["integer", "string"],
```

Comment:
@deubaka can we reflect this change in the [bid-param section of the docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/visx.md#bid-params) please?

Reply:
Good catch, thanks! We've added the corresponding updates in this PR, https://github.com/prebid/prebid.github.io/pull/4273.